library/src/components/Bindings.tsx

```diff
@@ -10,13 +10,28 @@ interface Props {
 }
 
 export const Bindings: React.FunctionComponent<Props> = ({
-  name = 'Bindings',
+  name = 'Binding',
```

Comment:
What about using the `specific information ` suffix as text (validated with @derberg in the original implementation). This provides a more comprehensive label for non-tech which are reading the generated documentation ;

Reply:
I changed it and it's an example how it looks:

![image](https://user-images.githubusercontent.com/20404945/130444318-e1a0c4dc-b530-4a82-bc6b-693f5a5416fd.png)

Reply:
very nice 👌